frameworks/digital-outcomes-and-specialists/manifests/edit_brief.yml

```diff
@@ -57,128 +57,102 @@
-    - which nice-to-have skills and experience they have
-
-    You can shortlist the specialists who:
-
-    - can start work when you need them to
-    - are within your budget
-    - meet all the essential skills and experience
-    - provide the most nice-to-have skills and experience
-
-    [Read about how to shortlist.](https://www.gov.uk/guidance/how-to-shortlist-digital-outcomes-and-specialists-suppliers)
-  questions:
-    - essentialRequirementsSpecialists
-    - niceToHaveRequirementsSpecialists
-    - numberOfSuppliersSpecialists
--
-  name: Shortlist criteria
-  editable: True
-  step: 2
-  description: |
-    After you’ve published your requirements, suppliers will apply and tell you:
-
-    - if they have the essential skills and experience
-    - which nice-to-have skills and experience they have
-
-    You can shortlist the suppliers who:
+    You may need to ask for evidence of skills and experience if you have too many suppliers.
 
-    - meet all the essential skills and experience
-    - provide the most nice-to-have skills and experience
 
-    [Read about how to shortlist.](https://www.gov.uk/guidance/how-to-shortlist-digital-outcomes-and-specialists-suppliers)
-  questions:
-    - essentialRequirementsParticipants
-    - niceToHaveRequirementsParticipants
-    - numberOfSuppliersParticipants
+    **3\. Evaluate your shortlist**<br />
+    Ask the shortlisted suppliers for: 
 
--
-  name: Evaluation criteria
-  editable: True
-  step: 2
-  description: |
-    You need to tell suppliers how you’re going to evaluate them when you publish your requirements.
+      - evidence of their skills and experience
+      - a proposal explaining how they plan to meet your requirements
 
-    To find the most suitable supplier, you must evaluate your shortlist on:
+    You must tell suppliers how you’ll evaluate and assess them.
 
-    - technical competence, using essential skills and experience, nice-to-have skills and experience, and their proposal
-    - ‘cultural fit’, eg how the supplier will work in your organisation
-    - the price of the proposal
   questions:
+    - numberOfSuppliersOutcomes
     - weightingOutcomes
-    - essentialRequirementsOutcomes
-    - niceToHaveRequirementsOutcomes
-    - successCriteriaOutcomes
+    - technicalCompetenceCriteriaOutcomes
     - culturalFitCriteriaOutcomes
     - priceCriteria
     - evaluationTypeOutcomes
 
 -
-  name: Evaluation criteria
+  name: Shortlist and evaluation criteria
   editable: True
   step: 2
   description: |
-    You need to tell suppliers how you’re going to evaluate them when you publish your requirements.
+    There are 3 stages to finding the specialist that best meets your needs.
+
+    **1\. View specialists who applied**<br />
+    When specialists apply, they declare which of your ‘essential’ and ‘nice&#8209;to&#8209;have’ skills and experience they have. Specialists who don’t meet your essential skills and experience are excluded.
```

Comment:
OK, having looked through the rest of the content a bit I don't think we need the scare quotes round `essential` and `nice-to-have` where I've mentioned them?  Seems weird to have it just for these three sentences but not anywhere else.